packages/firestore/test/unit/specs/perf_spec.test.ts

```diff
@@ -0,0 +1,216 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Query } from '../../../src/core/query';
+import { doc, orderBy, path } from '../../util/helpers';
+
+import { describeSpec, specTest } from './describe_spec';
+import { spec } from './spec_builder';
+
+const STEP_COUNT = 10;
+
+describeSpec('Performance Tests:', ['benchmark'], () => {
+  specTest('Insert a new document', [], () => {
```

Comment:
Could you include ${STEP_COUNT} in the description of tests to accurately reflect what's being benchmarked?  With the results in your PR description I was disappointed how slow everything was, but knowing that your doing 10 reps makes me feel a fair amount better.

Similarly, the results (e.g. when copy/pasted into an email or PR description) would be more meaningful if the descriptions were more explicit / precise as to what's being measured.  E.g.: Write document and handle server acknowledgement [repeat ${STEP_COUNT} times]

Reply:
I added it to the test name via the description in `describe`.